### PR TITLE
1281 rabbitmq cluster in certificates domain

### PR DESCRIPTION
**Pull request by Exobitt**

### CODE REVIEW

File: domains/certificates/chart/values.yaml [hunk 1]
```diff
@@ -7,3 +7,14 @@ postgres:
     name: sibedge/postgres-plv8
     tag: 15.0-3.1.4-alpine
   size: 10Gi
+rabbitmq-cluster-operator:
+  extraDeploy:
```
Comment (duizer) [inline domains/certificates/chart/values.yaml:11]: Do we want to define the cluster using "extraDeploy"? If possible I would prefer to have it in its own YAML-file in templates/ folder.

File: domains/certificates/chart/values.yaml [hunk 2]
```diff
@@ -7,3 +7,14 @@ postgres:
...
+    - apiVersion: rabbitmq.com/v1beta1
+      kind: RabbitmqCluster
+      metadata:
+        name: certificate-rabbitmq-cluster
+      spec:
+        replicas: 1
+        rabbitmq:
+          additionalConfig: |
+            log.console.level = debug
```
Comment (duizer) [inline domains/certificates/chart/values.yaml:20]: Not sure that we want to have this log.console.level. I suggest that it is removed and we just use the defaults

File: domains/certificates/chart/values.yaml (hunk 1 above)
Comment (Exobitt) [reply domains/certificates/chart/values.yaml:11]: Marked as resolved, as this pr is still a draft and not done yet.

File: domains/certificates/chart/values.yaml (hunk 2 above)
Comment (Exobitt) [reply domains/certificates/chart/values.yaml:20]: Marked as resolved, as this pr is still a draft and not done yet.

**Pull request closed without merging**